pkg/codegen/docs_integration_test.go

```diff
@@ -318,6 +319,49 @@ func TestGetLanguageTypeString(t *testing.T) {
 	}
 }
 
+func BenchmarkGetPropertyNames(b *testing.B) {
```

Comment:
Actually, would it work to move this to its own file that had a `//go:build go1.24` compiler directive?

Reply:
Is `b.Loop()` what requires Go 1.24 here? Could this loop on `b.N` instead?

Reply:
`b.Loop()` is what requires 1.24.